src/libmongoc/tests/unified/entity-map.c

```diff
@@ -977,6 +1003,18 @@ entity_map_get_changestream (entity_map_t *entity_map,
    return (mongoc_change_stream_t *) entity->value;
 }
 
+mongoc_cursor_t *
+entity_map_get_cursor (entity_map_t *entity_map,
+                       const char *id,
+                       bson_error_t *error)
+{
+   entity_t *entity = _entity_map_get_by_type (entity_map, id, "cursor", error);
```

Comment:
Seems odd to me that string literals are used for entity types. I filed [CDRIVER-4073](https://jira.mongodb.org/browse/CDRIVER-4073) for that, but it's surely outside the scope of this PR.